src/Uno.UI/UI/Xaml/FrameworkTemplate.cs

```diff
@@ -40,8 +40,19 @@ protected FrameworkTemplate()
 			=> throw new NotSupportedException("Use the factory constructors");
 
 		public FrameworkTemplate(Func<View?>? factory)
-			: this(null, _ => factory?.Invoke())
+			: this(null, (o, s) =>
+			{
+				s.IsIgnored = true;
```

Comment:
check in v1 or v2 for what was this used for?

Reply:
first appeared in v2
nothing using this property is found in the commits, possibly never committed or rebased over...
prior to moving to source-gen setup, we were injecting tp for all visual-descendants before exiting LoadContent
this property could be either used to:
a. block tp injection between control/and its template members; or
	^ which proven to be incorrect, since that cant handle interwoven templates
b. block tp injection from specific class, eg: DataTemplate (as opposite to ItemsPanelTemplate or ControlTemplate)
	^ handled in their LoadContent impl